test_common/jest.config.js

```diff
@@ -37,27 +15,17 @@ const config = {
       statements: 100,
     },
   },
-  moduleDirectories: ['node_modules', 'client', 'server'],
-  moduleFileExtensions: ['js', 'feature', 'ts'],
-  testEnvironment: 'jsdom',
-  testMatch: ['**/*.spec.ts', '**/*.steps.ts'],
-  testPathIgnorePatterns: ['/e2e/'],
-  transform: {
-    '^.+\\.(ts|tsx)$': 'ts-jest',
-    '^.+\\.(js|jsx|ts|tsx)$': [
-      'babel-jest',
-      { configFile: './test_common/test.babel.js' },
-    ],
-  },
-  moduleNameMapper: {
-    ...jestModuleMapper,
-  },
-  setupFiles: ['<rootDir>/test_common/jest_cucumber_support/index.ts'],
-  globals: {
-    'ts-jest': {
-      tsconfig: '<rootDir>/test_common/tsconfig.test.json',
-    },
-  },
+  collectCoverageFrom: [
+    '(client|server)/**/*.{js,ts,jsx,tsx}',
```

Comment:
By default, jest gets coverage from all files that have been tested (and files it imports). This also includes files without tests (so that coverage drops to reflect missing testing).